heidegger_index/views.py

```diff
@@ -90,7 +95,21 @@ def get_context_data(self, **kwargs):
         lemma = context["lemma"]
         context["children"] = lemma.children.all()
         context["related"] = lemma.related.all()
+        try:
+            lemma_urn = CTS_URN(lemma.urn)
+            if lemma.type == "w" and lemma_urn.passage_component:
+                p_link = 'https://scaife-cts.perseus.org/api/cts?request=GetPassage&urn=' + lemma.urn
+                p_response = requests.get(p_link)
+                parsed_xml = BeautifulSoup(p_response.text, 'html.parser')
```

Comment:
Ben zelf niet zo'n groot fan van BeautifulSoup, die wil nog wel eens wat wijzigen aan de input (dat is overigens bewust, het is ontworpen om met slechte HTML om te gaan). Omdat je XML terugkrijgt, kun je rigoreuzer te werk gaan met lxml: https://lxml.de/. Heb je dat overwogen?

Reply:
lxml vind ik nogal engimatisch om mee te werken. Gewoon de inhoud onder één `<p>` tag parsen bleek een uitdaging. Ik heb het voor nu zo gelaten maar dan in heidegger_index/models.py als functie van het lemma model.